test-clients/src/main/java/com/hedera/services/bdd/suites/autorenew/ContractAutoExpirySpecs.java

```diff
@@ -391,22 +403,29 @@ private HapiSpec storageExpiryWorksAtTheExpectedInterval() {
                                         ByteString.copyFromUtf8("As 'twere a spinning-top"))),
                         createLargeFile(GENESIS, initcode, literalInitcodeFor("InstantStorageHog")),
                         enableContractAutoRenewWith(minimalLifetime, 0),
-                        contractCreate(contractToRemove, 63)
+                        contractCreate(contractToRemove, new BigInteger("63"))
                                 .gas(2_000_000)
                                 .entityMemo("")
                                 .bytecode(initcode)
                                 .balance(0)
                                 .autoRenewSecs(minimalLifetime),
                         tokenAssociate(
                                 contractToRemove,
-                                List.of(aFungibleToken, bFungibleToken, nonFungibleToken)),
+                                List.of(
+                                        aFungibleToken,
+                                        bFungibleToken,
+                                        cFungibleTokenWithCustomFees,
+                                        nonFungibleToken)),
                         cryptoTransfer(
                                 moving(aFungibleAmount, aFungibleToken)
                                         .between(TOKEN_TREASURY, contractToRemove),
                                 moving(bFungibleAmount, bFungibleToken)
                                         .between(TOKEN_TREASURY, contractToRemove),
+                                moving(cFungibleAmount, cFungibleTokenWithCustomFees)
+                                        .between(TOKEN_TREASURY, contractToRemove),
                                 movingUnique(nonFungibleToken, 1L, 2L)
                                         .between(TOKEN_TREASURY, contractToRemove)),
+                        // TODO: would like to add a check that the token move was successful
```

Comment:
Is it possible / desirable to add this check? It seems that in order to check this, we'd have to wait for another round of consensus, significantly slowing this test down.

Reply:
You mean submitting another transaction is slowing down the test ?

Reply:
I added a check that the token move was successful, as we discussed.